WalletWasabi/Legal/Assets/LegalDocuments.txt

```diff
@@ -197,8 +197,8 @@ Personally Identifiable Information
 Handling Information
 ====================
 
-    Since we are working on privacy, our mission is to regain personal privacy, our Services are designed to be used without indication of any personal data. For this reason we do not have any kind of data collecting solutions built into our products. There may only be one personal data processing in our Service, for customer support in case of technical problems: visitors may, indicate their email addresses voluntarily to get notifications in case of any potential technical problems or other enquiries. These e-mail addresses are solely used to answer users’ questions and are erased after 100 days. In this case, the processing of the data is based on a freely given consent to Article 6 (1) (a) of the GDPR and is aimed at the effective handling of the complaint.
-    As the main platform for users’ technical questions and issues we use Reddit to answer all questions, and we do not retain any data that can be subsequently identified / associated with the user.
+    Since we are working on privacy, and our mission is to regain personal privacy, our Services are designed to be used without indication of any personal data. For this reason we do not have any kind of data collecting solutions built into our products. There may only be one personal data processing in our Service, for customer support in case of technical problems: visitors may, indicate their email addresses voluntarily to get notifications in case of any potential technical problems or other inquiries. These e-mail addresses are solely used to answer users’ questions and are erased after 100 days. In this case, the processing of the data is based on a freely given consent to Article 6 (1) (a) of the GDPR and is aimed at the effective handling of the complaint.
```

Comment:
GitHub shows that something is changed here, but I cannot tell what exactly...

Reply:
Before:
Since we are working on privacy, our mission is to regain personal privacy, our Services are designed to be used without indication of any personal data.
After:
Since we are working on privacy, **and** our mission is to regain personal privacy, our Services are designed to be used without indication of any personal data.